IPHONEX.md

```diff
@@ -0,0 +1,224 @@
+# iPhone X support
+
+The iPhone X is different to other iPhones in several ways.
+Its screen fills almost the entire front of the device,
+the screen size and aspect ratio are different (375×812 points or 1125×2436 pixels),
+it has distinctly rounded corners, 
+and there's the notch: the status bar is split either side of a cutout area that houses the front camera, earpiece and other sensors.
+All this impacts how iOS apps are shown on the screen, and how iOS should be built (see also [the Apple website](https://developer.apple.com/ios/update-apps-for-iphone-x/) for more info on this topic).
+
+Mendix apps will run out of the box on an iPhone X, but in order to use the full real-estate of the screen, a few customizations are needed.
+This documents explains what needs to be done.<sup>1</sup>
+
+## Configuring storyboard images in your Mendix hybrid app
+
+When building Mendix hybrid apps, there are three main flows to produce the installable package:
+
+1. By going through the `Mobile App` flow in the Mendix Portal and selecting `Build in the cloud`
+    
+    This is generally the fastest and easiest way to build your Mendix hybrid app.
+    Unfortunately, this flow does _not yet_ support the new storyboard images.
```

Comment:
Are we planning to implement this at one point? 
The not yet can be interpreted as a promise (to implement it).